### PR TITLE
Adding installer-provision-lb-ip changes for acc-provision-config

### PR DESCRIPTION
**Pull request by abhis2112**
Main file:
1. provision/acc_provision/templates/acc-provision-configmap.yaml

### CODE REVIEW

File: provision/testdata/.acirc [hunk 1]
```diff
@@ -1,2 +1,2 @@
-|public subnets|: [u'subnet-06cdb0b67bf86fcbf', u'subnet-0814cdfa1911664e7']
-|private subnets|: [u'subnet-093bf9479f50168f7', u'subnet-00a54d51ba268fe29']
+|public subnets|: ['subnet-06cdb0b67bf86fcbf', 'subnet-0814cdfa1911664e7']
```
Comment (snaiksat) [inline provision/testdata/.acirc:1]: we uncheckout this file

File: provision/testdata/base_case.kube.yaml [hunk 2]
```diff
@@ -1360,7 +1360,9 @@ data:
                     "external_networks": [
                         "default"
                     ]
-                }
+                },
+                "vmm_domain": {
+                 }
```
Comment (snaiksat) [inline provision/testdata/base_case.kube.yaml:1365]: i am wondering why the base case is being affected by this change

Comment (snaiksat) [reply provision/testdata/base_case.kube.yaml:1365]: i think i know...

**Comment by abhis2112** [comment]
Added in another commit

**Pull request closed without merging**